React-Native/en/1.4.4.md

```diff
@@ -1,4 +1,4 @@
-# Success Criterion 1.4.4 - Resize text
+# [Success Criterion 1.4.4 - Resize text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text)
 
 In React Native text scales automatically depending on the OS settings for font sizes. Also, all dimensions in React Native are unitless, and represent density-independent pixels.
```

Comment:
Code examples how to disable resize text?

Because I think quite a lot of developers disable it, by showing the code that does this, it is easier to remove it.